### PR TITLE
Vic metrics

### PR DESCRIPTION
**Pull request by lihanli**
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/8958

### CODE REVIEW

**Comment by rlbaker** [comment]
LGTM

File: app/uploaders/log_metrics.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  included do
+    after(:store, :log_metrics)
+  end
+
+  def log_metrics(file)
+    class_name = self.class.to_s.underscore
+    class_prefix = "#{KEY_PREFIX}#{class_name}"
+
+    %w[size content_type].each do |attr|
+      StatsD.measure("#{class_prefix}.#{attr}", file.public_send(attr))
```
Comment (patrickvinograd) [inline app/uploaders/log_metrics.rb]: I don't think doing `StatsD.measure(key, content_type_string)` will do anything useful - the arg is supposed to be a numeric value that statsd can aggregate/average/etc. We could change that to a counter and call increment on it, but I'd rather have the content type be a tag on the size measurement, so:

Comment (patrickvinograd) [reply app/uploaders/log_metrics.rb]:
```
StatsD.measure("#{class_prefix}.size", file.size, tags: ["content_type:#{file.content_type}"])
```
if my Friday afternoon ruby syntax is right

Comment (patrickvinograd) [reply app/uploaders/log_metrics.rb]: And actually, might be best to take just the second part of the content_type, so transform `image/gif` to `gif`, etc.

**Comment by lihanli** [comment]
@patrickvinograd ok i made the changes

**Pull request merged**